jetty-server/src/main/java/org/eclipse/jetty/server/handler/ErrorHandler.java

```diff
@@ -431,19 +431,23 @@ private void writeErrorJson(HttpServletRequest request, PrintWriter writer, int
     {
         writer
             .append("{\n")
-            .append("  url: \"").append(request.getRequestURI()).append("\",\n")
-            .append("  status: \"").append(Integer.toString(code)).append("\",\n")
-            .append("  message: ").append(QuotedStringTokenizer.quote(message)).append(",\n");
+            .append("  \"url\": \"").append(request.getRequestURI()).append("\",\n")
+            .append("  \"status\": \"").append(Integer.toString(code)).append("\",\n")
+            .append("  \"message\": ").append(QuotedStringTokenizer.quote(message)).append(",\n");
```

Comment:
here too, probably some of the libraries will ignore this?!

Reply:
@gbrehmer good catch I guess many librairies will ignore but better to fix it